docs/cli-how-to.md

```diff
@@ -1,3 +1,5 @@
+<!-- I think cli must be after "AEA quick start" and before "Developer Guide" or between "Step by Step" and "Skills"-->
```

Comment:
I think it works where it is now. The Step by step guide references it in case someone wants to skip forward. Please remove comment.